src/test/java/io/confluent/idesidecar/restapi/cache/ClusterCacheTest.java

```diff
@@ -1,28 +1,11 @@
 package io.confluent.idesidecar.restapi.cache;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.verifyNoInteractions;
-import static org.mockito.Mockito.when;
-
-import io.confluent.idesidecar.restapi.cache.ClusterCache;
 import io.confluent.idesidecar.restapi.connections.ConnectionState;
 import io.confluent.idesidecar.restapi.connections.ConnectionStates;
 import io.confluent.idesidecar.restapi.exceptions.ClusterNotFoundException;
 import io.confluent.idesidecar.restapi.exceptions.ConnectionNotFoundException;
 import io.confluent.idesidecar.restapi.models.ConnectionSpec;
-import io.confluent.idesidecar.restapi.models.graph.CCloudEnvironment;
-import io.confluent.idesidecar.restapi.models.graph.CCloudGovernancePackage;
-import io.confluent.idesidecar.restapi.models.graph.CCloudKafkaCluster;
-import io.confluent.idesidecar.restapi.models.graph.CCloudOrganization;
-import io.confluent.idesidecar.restapi.models.graph.CCloudSchemaRegistry;
-import io.confluent.idesidecar.restapi.models.graph.CloudProvider;
-import io.confluent.idesidecar.restapi.models.graph.ClusterEvent;
-import io.confluent.idesidecar.restapi.models.graph.KafkaCluster;
-import io.confluent.idesidecar.restapi.models.graph.RealCCloudFetcher;
-import io.confluent.idesidecar.restapi.models.graph.RealLocalFetcher;
+import io.confluent.idesidecar.restapi.models.graph.*;
```

Comment:
Please remove the wildcard import.

Reply:
removing! also did [this ](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports)so these shouldn't appear anymore